### DISCUSSION TITLE
In Face Detection process, is it possible to form/generate an image of person whose faces got matched either self/ matching with some other face model

### DISCUSSION THREAD

**Discussion by RoshanCubastion** [davisking/dlib#2703] [Q&A]
I stored known_face_encodings with the person's name when I was training the images earlier, when using face_recognition.compare_faces(known_face_encodings, face_encoding_to_check, tolerance=0.40), detection is done correctly, I am returning the list of person's name if multiple faces found, but I want to return the person's image as well with whom the face is matching with

**Comment by davisking** [comment]
You have to store the image somewhere.  The image itself is not encoded in
the feature vector.
